weather.js

```diff
@@ -16,14 +19,27 @@ class Forecast {
 async function getWeather (request, response){
   let lat = request.query.lat;
   let lon = request.query.lon;
-  
+  let key= lon + 'day';
+  if (cache[key] && Date.now() - cache[key].timestamp < (1000 * 10)){
+    console.log('cache hit, whether for the win');
+    response.send(cache[key].data);
+  } else {
+    console.log('cache miss, no weather today.')
+  }
```

Comment:
Really good work so far!

- If we look in task card 1 for Performance, it says, ` If no recent result is found ("cache miss"), query the API for a fresh result, add it to server memory with the time of retrieval, and return the fresh result.`. 
- So here, you are doing a good job of logging the response, but we need to also:
     - make another call using axios to grab the data
     - map over the data result and then send the Forecast data array to the client
     - add it to server memory /add it to the cache 👍